finetune/datasets/multinli.py

```diff
@@ -0,0 +1,50 @@
+import os
+import logging
+from pathlib import Path
+
+import requests
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+from finetune import Entailment
+from finetune.datasets import Dataset
+
+logging.basicConfig(level=logging.DEBUG)
+
+FILENAME = "multinli.dev.csv"
+DATA_PATH = os.path.join('Data', 'Entailment', FILENAME)
+
+
+class MultiNLI(Dataset):
+
+    def __init__(self, filename=None, **kwargs):
+        super().__init__(filename=(filename or DATA_PATH), **kwargs)
+
+    def download(self):
+        """
+        Download Stanford Sentiment Treebank to enso `data` directory
+        """
+        path = Path(self.filename)
+        if path.exists():
```

Comment:
Personally, for a demo like this. I think its nice to do something like
 ```
if path.exists() and hashlib.md5(path).hexdigest() == CHECKSUM:
...
```

Just to mean that if it stops mid-download people don't get terrible results from half a csv, or worse have to dig around finding half a file to manually remove. Just an idea - not critical at all.